quoridor-app/app/component/board.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { View, StyleSheet } from "react-native";
+
+import Pawn from "./pawn";
+import Square from "./square";
+
+const styles = StyleSheet.create({
+  board: {
+    width: 360,
+    height: 360,
+    backgroundColor: "rgb(133, 65, 52)"
+  }
+});
+
+const DELTA = 40;
+const BOARD_PADDING = 5;
+const PAWN_TOP_PADDING = BOARD_PADDING - 6;
+const PAWN_LEFT_PADDING = BOARD_PADDING + 4;
+const PAWN_COLORS = ["rgb(217, 54, 59)", "rgb(184, 88, 46)"];
+
+const Board = ({ squares, pawns }) => {
+  if (!squares) {
+    return null;
+  }
+
+  return (
+    <View style={styles.board}>
+      {squares.map((item, key) => (
+        <Square
+          key={key}
```

Comment:
it seems to me that it is not a good practice to use the map `index` as a `key` ... https://menubar.io/react-keys-index. For example, from line 30, you will generate identical keys